src/backend/base/langflow/components/cleanlab/cleanlab_evaluator.py

```
@@ -0,0 +1,145 @@
+
+
+class CleanlabEvaluator(Component):
+    """A component that evaluates the trustworthiness of LLM responses using Cleanlab.
+
+    This component takes a prompt and response pair, along with optional system instructions,
+    and uses Cleanlab's evaluation algorithms to generate a trust score and explanation.
+
+    Inputs:
+        - system_prompt (MessageTextInput): Optional system-level instructions prepended to the user prompt.
+        - prompt (MessageTextInput): The user's prompt or query sent to the LLM.
+        - response (MessageTextInput): The response generated by the LLM to be evaluated. This should come from the 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)